service/src/test/java/bio/terra/cbas/pact/EcmPactTest.java

```diff
@@ -34,10 +33,9 @@ private void initEcmService(MockServer mockServer) {
     ecmService = new EcmService(ecmClient, new BearerToken("accessToken"));
   }
 
-  @Pact(consumer = "cbas", provider = "ecm")
+  @Pact(consumer = "cbas", provider = "externalcreds")
```

Comment:
Out of curiosity, was this changed because Pact broker expects `externalcreds` instead of `ecm`?